Agreements/Internal/IoM_Wal.txt

```diff
@@ -0,0 +1,9 @@
+COPX:*:*:IBRAR:EGCC:*:Isle of Man:Wallasey:*:17000:IBRAR
+COPX:*:*:PENIL:EGCC:*:Isle of Man:Wallasey:*:17000:PENIL
+COPX:*:*:MALUD:EGCC:*:Isle of Man:Wallasey:*:17000:MALUD
+
+COPX:*:*:*:EGGP:*:Isle of Man:Wallasey:*:13000:|
```

Comment:
With your permission, could we role out the fix-less climbing/descending arrows elsewhere perhaps?

Reply:
I have added a fix to anchor the agreement for now, to discuss later whether to give a purely descending arrow in future